File: 02-Comandos de entrada e saída/C/Pratica_01/main.c
#include <stdio.h>

int main() {

    printf("Digite o seu nome: "); // saída padrão. Impressão no monitor

    char user[50]; // local que armazena um nome com, no máximo, 50 caracteres.
    scanf("%s",&user); // entrada padrão. capturar o que foi digitado pelo usuário no teclado.

    printf("Ola, %s. Seja bem vindo!",user);
    return 0;
}

File: 02-Comandos de entrada e saída/Java/Pratica_01/Main.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		
		System.out.println("Digite o seu nome:"); // saída padrão. impressão no monitor
		
		String user; // local que armazena um nome
		Scanner reader = new Scanner(System.in); // criação de objeto para ler o teclado. Específico do Java
		user = reader.nextLine(); // entrada padrão. captura o que foi digitado pelo usuário no teclado

		System.out.println("Ola, " + user + ". Seja bem-vindo!");
	}
}